environments/python3/builder/defaultBuildCmd

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+pip3 install -r ${SRC_PKG}/requirements.txt -t ${SRC_PKG} && cp -r ${SRC_PKG} ${DEPLOY_PKG}
```

Comment:
I think the builder should also load the user function, so that if it has syntax errors, we'll catch them in the build output. (I'm fine with doing that in a separate PR.)